patches/@mui__internal-test-utils.patch

```diff
@@ -0,0 +1,34 @@
+diff --git a/build/createRenderer.js b/build/createRenderer.js
+index 800cf1520c1457d76c2eba097e7ebcb86c28de81..be0cfd6a08cf077bef2dbcd0dbb934c3d8763c06 100644
+--- a/build/createRenderer.js
++++ b/build/createRenderer.js
+@@ -211,7 +211,8 @@ function renderToString(element, configuration) {
+         },
+     };
+ }
+-const isVitest = 
++const isVitest =
++    ('vi' in globalThis && vi != null) ||
+ // VITEST is present on the environment when not in browser mode.
+ process.env.VITEST === 'true' ||
+     // VITEST_BROWSER_DEBUG is present on vitest in browser mode.
+diff --git a/src/createRenderer.tsx b/src/createRenderer.tsx
+index 96d80d00c18a50f7e35c7a1bcba9e19314904e08..8927248ae33c908601184cdceac99f1060a14b73 100644
```

Comment:
Did you make these changes to the material package as well? They can probably be done there without issues

Reply:
We can remove this patch. I've opened a PR [here](https://github.com/mui/material-ui/pull/47141)